source/manual/request-fastly-tls-certificate.html.md

```diff
@@ -10,7 +10,7 @@ related_applications: [bouncer, transition]
 When transitioning HTTPS domains, these are the steps to request a TLS
 certificate from Fastly.
 
-1. Use the 2nd line account to login to Fastly.
+1. Login to Fastly.
```

Comment:
How does a new person go about getting a Fastly account?

Reply:
Good point. Seems its fairly standard to say "speak to a member of the senior tech team" so I'll add that